README.md

```diff
@@ -47,113 +47,163 @@ If you find `googler` useful, please consider donating via PayPal.
 
 `googler` requires Python 2.7.x or Python 3.x to work.
 
-1. If you have git installed (the steps are tested on Ubuntu 14.04.3 x64_64):  
-<pre>$ git clone https://github.com/jarun/googler/  
-$ cd googler  
-$ sudo make install</pre>  
-To remove, run:  
-<pre>$ sudo make uninstall</pre>
+## Installing from this repository
+
+If you have git installed:
+
+    $ git clone https://github.com/jarun/googler/
+    $ cd googler
+
+Otherwise, download the latest [stable release](https://github.com/jarun/googler/releases/latest) or [development version](https://github.com/jarun/googler/archive/master.zip"), unpack the archive, and cd into the directory.
+
+Next, run
+
+    $ make install
+
+which installs to `/usr/local` by default. You might need to prepend `sudo` to the command above if `/usr/local` is only writable by root. To install to a different prefix, run
+
+    $ PREFIX=/path/to/prefix make install
+
+To remove, run
+
+    $ make uninstall
+
+Again, you might need `sudo` and/or `PREFIX=`.
+
+### Don't feel like installing to a prefix?
+
+`googler` is a standalone executable. From the containing directory:
+
+    $ ./googler
+
```

Comment:
I don't quite understand this because the links are already given above; moreover, the download options should be independent from whether installing to a prefix or running from the repo dir — actually that's the most important reason I restructured the installation section. Anyway I (hopefully) clarified the logical relations between steps even more in the latest version: https://github.com/zmwangx/googler/tree/readme-refresh#installing-from-this-repository.